module-2/backend/week_one.md

```diff
@@ -5,30 +5,109 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+* GET - A basic request to retrieve a page from the server
+* PUT - Client is sending information to the server
+* POST - 
+* PATCH - Updates Information but does not overwrite whole record
+* DELETE - Destroys Information from database
+
 2. What is Sinatra?
+
+* A very painful framework for local development and spinning up a local server.
+* Allows us to make web pages.
+
 4. What is MVC?
+
+* Models, Views, Controllers
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+* So they don't break.
+* It's easier to work with for all developers involved
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+* Instance Varibales
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+  ```ruby
+  get '/horses' do
+    @name = "Mr. Ed"
+    erb :index
+  end
+  ```
+  
 9. What's the purpose of ERB?
+
+* It's a html language page that allows the use of Embeded Ruby Code
+
 10. Why do I need a development AND test database?
+
+* Becasue we don't want to effect our development database information when building tests and running them
+* We do not want the database cleaner to destroy data in our production database
+
 11. What's responsive design?
+
+
 12. What is CRUD and why is it important?
+
+* Create, Read, Update and Delete. These are the tools needed to build restful routes.
+
 13. What does HTTP stand for? 
+
+* Hyper Text Transfer Protocol
+
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
+
+* Using ERB tags

```

Comment:
<%= ruby %> <% ruby %> 

one shows the information on the page, the other does not